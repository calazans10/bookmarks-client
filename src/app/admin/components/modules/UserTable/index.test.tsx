import React from 'react';
import { render } from '@testing-library/react';
import { users } from 'fixtures';
import { UserTable } from './';

describe('UserTable', () => {
  it('renders without crashing', () => {
    const { container } = render(
      <UserTable users={users.filter(user => !user.is_admin)} />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
