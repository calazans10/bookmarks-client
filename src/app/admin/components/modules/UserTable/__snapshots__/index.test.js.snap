// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserTable renders without crashing 1`] = `
<Table>
  <tbody>
    <TableRow
      key="e4f262c4-8dd3-4db4-85c8-83e03b8ecad4"
    >
      <TableColumn
        hasActions={false}
        label="Name"
      >
        John Doe
      </TableColumn>
      <TableColumn
        hasActions={false}
        label="Email"
      >
        john.doe@example.com
      </TableColumn>
    </TableRow>
  </tbody>
</Table>
`;
